src/main/webapp/js/instructorFeedbackEdit/customFeedbackPaths.js

```diff
@@ -51,12 +51,13 @@ var CustomFeedbackPaths = {
         // Empty string added to provide an empty option in spreadsheet dropdown
         // It prevents a feedback participant from being selected upon clicking away from dropdown
         CustomFeedbackPaths.allPossibleFeedbackGivers = [''];
-        CustomFeedbackPaths.allPossibleFeedbackGivers =
-                CustomFeedbackPaths.allPossibleFeedbackGivers.concat(CustomFeedbackPaths.studentEmails);
-        for (var i = 0; i < CustomFeedbackPaths.instructorEmails.length; i++) {
-            if (!CustomFeedbackPaths.allPossibleFeedbackGivers.includes(CustomFeedbackPaths.instructorEmails[i])) {
-                CustomFeedbackPaths.allPossibleFeedbackGivers.push(CustomFeedbackPaths.instructorEmails[i]);
-            }
+        var i;
+        for (i = 0; i < CustomFeedbackPaths.studentEmails.length; i++) {
+            CustomFeedbackPaths.allPossibleFeedbackGivers.push(
+                    CustomFeedbackPaths.studentEmails[i] + ' (Student)');
+        }
+        for (i = 0; i < CustomFeedbackPaths.instructorEmails.length; i++) {
+            CustomFeedbackPaths.allPossibleFeedbackGivers.push(CustomFeedbackPaths.instructorEmails[i] + ' (Instructor)');
```

Comment:
There's some danger in using `(Student)` and `(Instructor)` as indicators that these are students or instructors, since there's nothing stopping a team from getting named with the suffix `... (Student)`